app/views/static_pages/about.html.slim

```diff
@@ -0,0 +1,2 @@
+h1 About
+p Information about our project
```

Comment:
Please, insert here an emty line at the end of file

Reply:
Update, please review